config/manifests/bases/submariner.clusterserviceversion.yaml

```diff
@@ -48,61 +48,67 @@ spec:
     For complete information about subctl, please refer to [this page](https://submariner.io/operations/deployment/subctl).
     In addition to Operator and subctl, Submariner also provides [Helm Charts](https://submariner.io/operations/deployment/helm).
 
-    #### Deploying with subctl
-    Install the tool:
-    ```bash
-    curl -Ls https://get.submariner.io | bash
-    export PATH=$PATH:~/.local/bin
-    echo export PATH=\$PATH:~/.local/bin >> ~/.profile
-    ```
-    Deploy the broker:
-
-    `subctl deploy-broker --kubeconfig <PATH-TO-KUBECONFIG-BROKER>`
-
-    For each cluster you want to join, issue the following command:
-
-    `subctl join --kubeconfig <PATH-TO-JOINING-CLUSTER> broker-info.subm --clusterid <ID>`
-
-    More information can be found [here](https://submariner.io/operations/deployment/subctl).
-
-    Submariner is deployed using the Submariner CR. The following values can be specified:
-    * **namespace**: the namespace to install in;
-    * **serviceCIDR**: the service CIDR;
-    * **clusterCIDR**: the cluster CIDR;
-    * **token**: the Submariner token
-        to retrieve the token run the following command on the broker cluster, assuming the broker is running in the
-        *submariner* namespace replace the two *submariner* instances as appropriate
-
-        `kubectl -n submariner get secrets -o
-        jsonpath="{.items[?(@.metadata.annotations['kubernetes.io/service-account.name']=='submariner-client')].data.token}"
-        | base64 --decode`
-
-    * **clusterID**: the Submariner cluster ID;
-    * **colorCodes**: the Submariner color code(s) for this cluster;
-    * **debug**: *'true'* to enable logging debugging information;
+    On the broker cluster:
+
+    1. Create the broker namespace `submariner-k8s-broker`.
+    2. Install the submariner operator from the catalog.
+    3. Create a `Broker` CR.
+
+    On connected clusters:
+
+    1. Create the broker namespace `submariner-k8s-broker`.
```

Comment:
@mangelajo you are right but the reason we have to do it is because the bundle includes the broker manifests as well and when the users installs the operator on any cluster OLM will automatically deploy all manifests including the broker's. There is no way to add a condition when using this approach. As you mentioned in the previous comment, if the operator handles the namespace and the rest of the broker resources then we don't need this step.

Reply:
The user should now manually apply the broker manifests.
The description was updated accordingly.